app.js

```diff
@@ -9,18 +9,30 @@ var handlebars = require('express-handlebars')
 
 var app = express();
 
+// Handlebars helpers.
+var handlebarsConfig = handlebars.create({
+  helpers: {
+      json: function(data) { return JSON.stringify(data); }
+  }
+});
+
 // Environments configs. 
 app.set('port', process.env.PORT || 3000);
 app.set('views', path.join(__dirname, 'views'));
-app.engine('handlebars', handlebars());
+app.engine('handlebars', handlebarsConfig.engine);
 app.set('view engine', 'handlebars');
 app.use(express.urlencoded());
 app.use('/static', express.static('public'));
 
-// Routes.
-const discover = require('./routes/discover')
+// Load API keys.
+if (process.env.NODE_ENV !== 'production') {
+  require('dotenv').config();
+}
```

Comment:
I think using dotenv will require some configuration on production, since currently our Cloud Build pipeline does not know the API keys we are using. 

I will take this AI to config GCP cloud build to read API keys on build-time without putting the keys on Github. 

Filed #36 for tracking this.